en/about.html

```diff
@@ -44,7 +45,91 @@
         </div>
     </nav>
 
-    <main class="bg-gray-100"></main>
+    <main>
+        <div class="bg-gray-100 w-screen items-center flex flex-col">
+            <h3 class="uppercase font-bold text-center text-2xl mb-2 relative z-20 mt-2">WHO WE ARE.</h3>
+            <span class="bg-midnight h-[0.18rem] w-60 mb-4"></span>
+            <p class="uppercase font-medium text-center mb-2 mt-2"> RESTAURANT OWNERS, GROCERY STORES, CHEFS, FARMS, CATERERS, CORPORATIONS, RETIREES, 
+                STUDENTS, MOMS, DADS, KIDS. </p>
+            <div class="mt-10">
+                <img class=" mx-auto mb-8 max-h-7/12" src="../assets/abt-1.png" alt="Helpers distributing boxes of fruit">
+            </div>
+            <button id="about-btn-1" class="mb-20 bg-blue-500 hover:bg-blue-400 text-white font-bold py-2 px-4 border-b-4 border-blue-700 hover:border-blue-500 rounded">GET INVOLVED</button>
+        </div>
+
+        <div id="abt-pg-2" class="grid items-center md:grid-cols-[repeat(2,_1fr)] w-full relative mb-0">
+            <div class="flex flex-col items-center text-center">
+                <p class="mb-5 text-2xl font-normal text-black uppercase font-bold w-[70%]">HOW we help</p>
+                <p class="text-2xl font-normal text-midnight uppercase font-bold w-[70%]">make a difference.</p>
+                <span id="this-span" class="bg-midnight opacity-70 h-10 absolute bottom-0 left-0 z-0"></span>
+            </div>
+            <img class="shadow-2xl rounded-tr-md" src="../assets/joel-muniz-A4Ax1ApccfA-unsplash 2.png" alt="An assortment of vegetables">
+        </div>
+        <div class="class='parent'">
```

Comment:
Any reason why it has to be like this? Just checking
```suggestion
<div class="parent">
```